packages/expo/bundledNativeModules.json

```
@@ -88,7 +88,7 @@
   "react-native-web": "~0.19.6",
   "react-native-gesture-handler": "~2.14.0",
   "react-native-get-random-values": "~1.8.0",
-  "react-native-maps": "1.8.0",
+  "react-native-maps": "0.0.0",
```

This doesn't seem correct 🤔 

```suggestion
  "react-native-maps": "1.10.0",
```

---

yeah i'm not sure why uvm changed to 0.0.0 🤔

---

Looks like they now have `0.0.0` in package.json 😅 
https://github.com/react-native-maps/react-native-maps/blob/4292c870fc7c82f1899a3e4a38f34912967d33d1/package.json#L8